data/notebooks/explore.py

```diff
@@ -13,7 +13,7 @@
 bw2data.projects.set_current("Ecobalyse")
 os.chdir("/home/jovyan/ecobalyse/data")
 
-databases = list(bw2data.databases.keys())
+databases = list(bw2data.databases)
```

Comment:
Out of pure curiosity, just let me ask:

![](https://media.tenor.com/tDQXUq-HakUAAAAM/why-just-why.gif)